src/FixMath.h

```diff
@@ -503,6 +503,15 @@ class UFix
       @return The floating point value.
   */
   float asFloat() const { return (static_cast<float>(internal_value)) / (next_greater_type(1)<<NF); }
+
+    /** Return the integer part of the number, as a standard C type integer (uint8_t, uint16_t etc) depending on NI.
+      @return The integer part, as a C integer type.
+  */
+ typename IntegerType<FixMathPrivate::uBitsToBytes(NI)>::unsigned_type asInt() const
```

Comment:
Yes, this addition make a lot of sense to me.

Reply:
Thanks, any preference between `asInt` and `asInteger`? I was hesitating as the second one might be less ambiguous with the `int` type. But it is also longer…

Reply:
No clear preference from my side. On very first sight, before I read the declaration, I somehow thought it possible, it might return a UFix/SFix truncated to the integer part. So that's another potential ambiguity, and I guess whichever name we choose, I will not be possible to fully avoid all ambiguity.